storage/flux/table.gen.go

```diff
@@ -182,7 +182,13 @@ func (t *floatGroupTable) advanceCursor() bool {
 		if typedCur, ok := cur.(cursors.FloatArrayCursor); !ok {
 			// TODO(sgc): error or skip?
 			cur.Close()
-			t.err = errors.Errorf("expected float cursor type, got %T", cur)
+			t.err = &influxdb.Error{
+				Code: influxdb.EInternal,
```

Comment:
This will report itself as "An internal server error occurred" just like the previous error. Is this intentional?

Reply:
No that is not intentional. Is that because `Msg` is nil?

Reply:
It's because of the code. The `EInternal` code indicates it is an internal error that shouldn't be shown to users.

Reply:
Should it be `Invalid` instead?

Reply:
Yes, but probably try to make it a more informed error message like "attempted to combine rows of different types" or something. Then use the internal error as the nested error on the `influxdb.Error` type.